src/posix/main.c

```diff
@@ -26,6 +26,8 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include "platform/platform-config.h"
```

Comment:
Only openthread-system.h is exported.

Reply:
I kept this include in new push since this file is not exported (is private) - and having all config definitions would be useful.